### DISCUSSION TITLE
[UI/UX 建议]：话题和助手 UI 修改建议

### DISCUSSION THREAD

**Discussion by DENGJIANGDI** [CherryHQ/cherry-studio#1870] [General]
如图所示。可以简化界面，专注对话。
简化多个对话间来回切换逻辑和速度。
建议增加一个翻译快捷键，输出结果直接在对话窗口弹出，妥妥的翻译器替代工具。
![image](https://github.com/user-attachments/assets/ddf5d06a-8f47-4248-b145-437c18ad0c0c)

**Comment by kangfenmao** [comment]
看看其他用户意见

**Comment by ZzzzRyan** [comment]
## 话题和助手

话题和助手的界面不建议合并，原因如下：

1. 有时一个助手下话题太多，展开后无法快速切换到其他助手；
2. 助手和话题本身不是同类事物，放在一个面板里会较为混乱。

## 设置

感觉这个设置面板里的 **Message Setting** 和 **Input Setting** 其实和 左下角<img src="https://github.com/user-attachments/assets/2e912237-95ce-4d81-b591-2ba62b4868c7" width="30">里的 **General Setting**、**Data Setting** 是同一层级的，这些可以放到一起（放到侧栏或者集中到哪感觉都可以）。

<img src="https://github.com/user-attachments/assets/ab622fd0-09d1-4631-92a6-591240941969" height="450">
<img src="https://github.com/user-attachments/assets/1d46d766-96c1-4d82-b85c-6980b29022a5" height="450">

而 Model Setting 是单独的、取决于当前助手的。保持在这里就可以。或者删掉，毕竟点击左上角助手名称也可以直接调整。

![image](https://github.com/user-attachments/assets/8fc20738-73d0-4d47-b3df-81ba3cdb1646)

## 其他

说到这，刚好还有一些相关建议。

比如，<img src="https://github.com/user-attachments/assets/2e912237-95ce-4d81-b591-2ba62b4868c7" width="30">里的 **Default Assistant** 可以去除掉，直接在 Agent 那里放一个默认的、无法删除的 Default 就行了，统一到一个地方。

![image](https://github.com/user-attachments/assets/61e00489-7b3f-4883-b284-58e5f785c966)

以及，这个Default Model也可以精简。

![image](https://github.com/user-attachments/assets/ae26d648-b757-43bc-80e5-8ddf0b38b866)

---

以上，供参考。

**Comment by DENGJIANGDI** [comment]
楼上朋友说的很有道理，会出现有较多话题的情况。可以参考openai官网的对话界面。
以下是我的改进建议：将话题放在助手下方，分成2块，太多滚动即可
![image](https://github.com/user-attachments/assets/f6d293ea-9013-4570-8381-4f4329e8b550)
